test/shortestpaths/astar.jl

```diff
@@ -10,7 +10,10 @@
             @inferred(a_star(g, 1, 4, d2))
         @test isempty(@inferred(a_star(dg, 4, 1)))
     end
-    @testset "GenericGraph and GenricDiGraph with SimpleEdge" for g in test_generic_graphs(g3), dg in test_generic_graphs(g4)
+    @testset "GenericGraph and GenricDiGraph with SimpleEdge" for g in
```

Comment:
this is all from the automatic formatter, right?

Reply:
i point this case out specifically, because it looks funky to me.

Reply:
yeah, it does look weird but it's JuliaFormatter's work
thanks for pointing it out!